proposals/ytt/005-semver/README.md

```diff
@@ -0,0 +1,217 @@
+---
+title: "Semver support in the standard library"
+authors: [ "Max Brauer <mbrauer@vmware.com>" ]
+status: "draft"
+approvers: [ ? ]
```

Comment:
**question:** Who should be listed as `approvers`?

Reply:
At a minimum I'd suggest @pivotaljohn and probably @cppforlife. John is the ytt approver, and DK is a general lead over all projects.

Reply:
Alrighty, I listed them. Ty @neil-hickey !